spring-integration-core/src/test/java/org/springframework/integration/handler/ServiceActivatorOnMockitoMockTests.java

```diff
@@ -82,7 +82,7 @@ public void shouldInvokeMockedSingleMethodOnClass() {
 	@SuppressWarnings("deprecation")
 	public static class SingleMethodAcceptingHeaderOnClass {
 
-		public String move(@org.springframework.integration.annotation.Header("s") String s) {
+		public String move(@org.springframework.messaging.handler.annotation.Header("s") String s) {
```

Comment:
If we do this, no need to have `@SuppressWarnings("deprecation")` above anymore.
As well as we can simply `import` this `@Header` class.
Will fix on merge.